src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/TeamCreation/TeamCreationSteps/Confirmation.tsx

```diff
@@ -44,15 +45,11 @@ const confirmationList = [
   t('teamCreationConfirmListItem3'),
 ];
 
-const errorTextMap = {
-  'user-already-in-a-team': t('teamCreationAlreadyInTeamError'),
-  'not-found': t('teamCreationUserNotFoundError'),
-};
+const alreadyPartOfTeamErrorCode = 403;
```

Comment:
We don't have a label for it? I imagine it's a new response

Reply:
We have a label for this error but I thought it is better to compare status code instead of label string.
As per api documentation, we only receive 403 for this case.
I am open to change it if you insist 💯

Reply:
maybe we could use this:

```
import {StatusCodes as HTTP_STATUS} from 'http-status-codes';
```

but just a suggestion